src/pages/reactivistas/index.tsx

```diff
@@ -1,26 +1,127 @@
 import Modal from '../../components/Modal';
+import { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { usePreviewSubscription } from '../../lib/sanity';
+import { reactGroupQuery } from '../../lib/querys';
+import Hero from '../../components/Hero';
 
-const ReactGroupPage: React.FC = () => {
+const ReactGroupPage: React.FC<
+  InferGetStaticPropsType<typeof getStaticProps>
+> = ({ data, preview }) => {
   const [isModalOpen, setIsModalOpen] = useState(false);
 
+  const { data: groups } = usePreviewSubscription(reactGroupQuery, {
+    initialData: data,
+    enabled: preview,
+  });
+
+  const onAddParticipantSubmit = async (
+    event: FormEvent<HTMLFormElement>,
+    discordUser: string,
+    id: string,
+  ) => {
+    event.preventDefault();
+
+    const data = {
+      discordUser: discordUser,
+      id: id,
+    };
+
+    try {
+      const res = await fetch('/api/add-participant', {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
   return (
     <Layout title="Iniciativas">
+      <Hero small title="Reactivistas" />
       <div className="pb-24 bg-indigo-100 sm:pt-10">
+        <div className="container mx-auto m-10 p-10 bg-white shadow">
+          <h1 className="text-2xl font-bold leading-7 text-black sm:text-2xl sm:leading-9 sm:truncate">
+            Sumate a los grupos que están comenzando
+          </h1>
+          <div className="flex flex-wrap m-10">
+            {groups.map((group: ReactGroup) => {
+              const [discordUser, setDiscordUser] = useState('');
+
+              return (
+                <Fragment key={group.name}>
+                  {group.status === 'approved' ? (
+                    <div className="flex flex-col flex-auto shadow-md m-5 p-10">
+                      <h3 className="font-medium leading-7 text-lg text-primary mb-5 sm:leading-9 sm:truncate">
+                        ⚛ {group.name}
+                      </h3>
+                      <ul className="mb-6">
+                        <li key={group.topic} className="mb-2 font-medium">
+                          Tema: <b>{group.topic}</b>
+                        </li>
+                        <li className="mb-2 font-medium text-primary">
+                          <a href={group.studyMaterial}>
+                            📚 Material de Estudio
+                          </a>
+                        </li>
+                        <li className="font-medium">
+                          Fecha de inicio: {group.startDate}
+                        </li>
+                      </ul>
+                      {group.participants && group.participants.length >= 10 ? (
+                        <div className="font-md text-md text-red-500">
+                          Grupo lleno
+                        </div>
+                      ) : (
+                        <form
```

Comment:
Would it be too much of a hassle to ask you to use `react-hook-form` so we are consistent with the rest of the forms that we have on the site?

Reply:
I tried to use `react-hook-form` but it had a strange behavior, only the last button (of the last ReactGroupCard) submitted. I guess that happens because of the way the library works with the rendering of multiple forms inside a map function.

Reply:
Ah, interesting. Good to know, let's keep it as it is then, we can review it later.